content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -504,6 +504,22 @@ stops adjusting the target (and sets the `ScalingActive` Condition on itself
 to `false`) until you reactivate it by manually adjusting the target's desired
 replica count or HPA's minimum replica count.
 
+## Recommended Deployment Configuration
```

Comment:
```suggestion
## Recommended Deployment configuration
```

Because `Deployment` is the proper name, I've chosen to leave it capitalized.  Let me know if this still is not preferred.